src/Tasks/RoslynCodeTaskFactory/RoslynCodeTaskFactoryCompilers.cs

```diff
@@ -99,7 +120,8 @@ protected override string GenerateFullPathToTool()
             }
 
 #if RUNTIME_TYPE_NETCORE
-            return DotnetCliPath;
+            Log.LogMessageFromText($"dotnet path is {dotnetCliPath}", StandardOutputImportanceToUse);
```

Comment:
Feels like this should be at most low-pri, and I question having it at all. Do we not log the command line we're going to invoke before we invoke it? If we do is that clear enough?